api/server/router/volume/volume.go

```diff
@@ -44,6 +44,14 @@ func (r *router) initRoutes() {
 			handlers.NewSchemaValidator(nil, schema.ServiceVolumeMapSchema, nil),
 		),
 
+		// get all volumes from all services with attachments
+		httputils.NewGetRoute(
```

Comment:
Ok will remove it. Was troubleshooting and I believe the actual missing
piece was the line above the query string for post handler.

On Tuesday, April 26, 2016, Schley Andrew Kutz <notifications@github.com>
wrote:

> In api/server/router/volume/volume.go
> <https://github.com/emccode/libstorage/pull/53#discussion_r61135159>:
>
> > @@ -44,6 +44,14 @@ func (r *router) initRoutes() {
> >  			handlers.NewSchemaValidator(nil, schema.ServiceVolumeMapSchema, nil),
> >  		),
> >
> > +		// get all volumes from all services with attachments
> > +		httputils.NewGetRoute(
>
> I'm not sure why this was added. It's unnecessary as the actual handler
> checks for the query string. Additionally, the same name cannot be used for
> two routes or one will overwrite the other.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/emccode/libstorage/pull/53/files/3159ffe2e4cf06547b2cfb4b889c54effdc1a3ee#r61135159>
>